app/content/profile/index.js

```diff
@@ -1,19 +1,54 @@
+import {
+  BottomSheetModal,
+  BottomSheetModalProvider,
+} from "@gorhom/bottom-sheet";
 import { useNavigation } from "expo-router";
-import { ScrollView, StyleSheet, Text, View } from "react-native";
+import { signOut as signoutFireBase } from "firebase/auth";
+import { useCallback, useMemo, useRef } from "react";
+import { Pressable, ScrollView, StyleSheet, Text, View } from "react-native";
 import { Appbar, PaperProvider } from "react-native-paper";
 import { SafeAreaView } from "react-native-safe-area-context";
 import DrillCard from "~/components/drillCard";
 import ProfileCard from "~/components/profileCard";
+import { useAuth } from "~/context/Auth";
 import drillData from "~/drill_data.json";
+import { auth } from "~/firebaseConfig";
 
 function Index(props) {
+  const { signOut } = useAuth();
   const navigation = useNavigation();
   const user = drillData["teams"]["1"]["users"]["1"];
```

Comment:
This user still depends on drillData, so the page still displays "John Doe"

Reply:
I saw you also have a PR to integrate profile with firebase (https://github.com/Golf-Drill-Challenge-App/Golf-App/pull/83), would you like me to copy in some stuff from your profile PR into my login PR for now?

Otherwise, the profile PR could be merged first (I can review it later too).

Reply:
Good point